reparo/metrics.go

```diff
@@ -67,7 +67,5 @@ func init() {
 	registry.MustRegister(queryHistogramVec)
 	registry.MustRegister(queueSizeGauge)
 
-	if gatherer, ok := registry.(prometheus.Gatherer); ok {
-		prometheus.DefaultGatherer = gatherer
-	}
+	prometheus.DefaultGatherer = registry
```

Comment:
Nit: When using the global registry, it can be simplified to `prometheus.MustRegister`.